### ISSUE TITLE
Doorbell not publishing MQTT messages

### ISSUE THREAD

**Issue by edwardhughes** [hjdhjd/homebridge-unifi-protect#182]
First, thank you for all of the work on the amazing plugin!

Second, I haven't submitted many github issues, so please forgive my noob-ness :)  I've been all over the documentation and searching through the code, so I apologize in advance if I've missed something obvious.

**Describe The Problem:**

My issue is that doorbell MQTT messages are no longer being published.  In fact, even with the feature option enabled my doorbell events aren't being logged.  The logs show that the MQTT broker connection is successful, and manually publishing to the topic causes my node-red automations to execute correctly.  I would expect this as I haven't changed them since initial development in late August.  Push notifications and the Homepod chime functions correctly, so it seems that doorbell events are being correctly detected by the plugin.  In fact, I can't find anything else with the plugin that doesn't work as I would expect.

The subscribed MQTT topic in my node-red flow is: unifi/protect/E063DAEF1FF9/doorbell

**To Reproduce:**

Press doorbell.  Watch logs and subscribe to expected MQTT topic.  See nothing in logs or on MQTT topic on doorbell event.  Push notifications and Homepod chime will notify of doorbell event.

**Logs:**

```
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO]: Connected to the Protect controller API (address: 192.168.0.1 mac: E263DACFD832).
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO]: Connected to the UniFi realtime update events API.
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO]: Discovered camera: South Front Yard [UVC G3] (address: 192.168.0.244 mac: E063DA3F1F19).
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO]: Discovered camera: UVC G3 Flex [UVC G3 Flex] (address: 192.168.0.240 mac: FCECDA1F503A).
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO]: Discovered camera: Doorbell [UVC G4 Doorbell] (address: 192.168.0.242 mac: E063DAEF1FF9).
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO]: Discovered camera: Nursery [UVC G3 Dome] (address: 192.168.0.243 mac: 7483C2CF4F3C).
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO]: Discovered camera: Front Porch [UVC G3 Dome] (address: 192.168.0.241 mac: 7483C2CF2A84).
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO]: Controller refresh interval set to 10 seconds.
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] South Front Yard [UVC G3]: Enabling motion sensor switch.
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] UVC G3 Flex [UVC G3 Flex]: Enabling motion sensor switch.
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Doorbell [UVC G4 Doorbell]: Smart motion detection enabled.
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Doorbell [UVC G4 Doorbell]: Enabling motion sensor switch.
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Nursery [UVC G3 Dome]: Enabling motion sensor switch.
[11/18/2020, 7:52:46 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Front Porch [UVC G3 Dome]: Enabling motion sensor switch.
[11/18/2020, 7:52:47 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Doorbell [UVC G4 Doorbell]: Enabling doorbell automation trigger.
[11/18/2020, 7:52:47 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Doorbell [UVC G4 Doorbell]: Discovered doorbell message switch: LEAVE PACKAGE AT DOOR.
[11/18/2020, 7:52:47 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Doorbell [UVC G4 Doorbell]: Discovered doorbell message switch: DO NOT DISTURB.
[11/18/2020, 7:52:47 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Doorbell [UVC G4 Doorbell]: Discovered doorbell message switch: BE RIGHT THERE!!.
[11/18/2020, 7:52:47 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Doorbell [UVC G4 Doorbell]: Discovered doorbell message switch: COME ON IN!.
[11/18/2020, 7:52:47 PM] [UniFi Protect] UDM Pro IV [UDM-PRO]: Connected to MQTT broker: mqtt://<username>:REDACTED@192.168.0.5 (topic: unifi/protect).
[11/18/2020, 8:04:18 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Doorbell [UVC G4 Doorbell]: HomeKit video stream request received from 192.168.0.34: 1280x720@30fps, 299 kbps. Using the 1600x1200@45fps (High) RTSP stream profile.
[11/18/2020, 8:04:21 PM] [UniFi Protect] UDM Pro IV [UDM-PRO] Doorbell [UVC G4 Doorbell]: Stopped video streaming session.
```

**Homebridge Configuration:**

```json
{
            "controllers": [
                {
                    "address": "192.168.0.1",
                    "mqttTopic": "unifi/protect",
                    "mqttUrl": "mqtt://<username>:<password>@192.168.0.5",
                    "password": "<password>",
                    "username": "<username>"
                }
            ],
            "options": [
                "Enable.Doorbell.Trigger.E063DAEF1FF9",
                "Enable.Motion.SmartDetect",
                "Enable.Motion.Switch",
                "Enable.LogDoorbell.E063DAEF1FF9"
            ],
            "platform": "UniFi Protect"
        }
```

**Environment:**

* **Homebridge Version:** v1.2.3
* **Node Version:** v14.15.0
* **Homebridge-UniFi-Protect Plugin Version:** 4.1.4
* **UniFi Protect Controller Version:** 1.15.0
* **UniFi Protect Controller Type:** UDMP
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** iOS 14
* **Operating System and OS Version:** Ubuntu 20.04.1 LTS

**Comment by hjdhjd** [comment]
Thanks for the report...I’ll take a closer look at this over the next few days.

Offhand, the obvious thing that I do see that’s incorrect is the log feature option you’re using - it has a typo. Take a closer look at the [feature option documentation for logging](https://github.com/hjdhjd/homebridge-unifi-protect/blob/master/docs/FeatureOptions.md#logging-feature-options). 😄 

The MQTT regression I’ll take a closer look at in the coming days.

**Comment by edwardhughes** [comment]
Dang!  Good catch on the typo.  After fixing that I am getting appropriate log entries on a doorbell event.

I'll keep troubleshooting what I better understand and update this issue if I uncover anything useful.

**Comment by hjdhjd** [comment]
I already have a sense of what the problem is...but if it makes you happy to hunt around, you do you. :smile:

**Comment by hjdhjd** [comment]
Fixed in v4.1.5.
